src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Dataplane/Azure.AnalysisServices.psd1

```diff
@@ -0,0 +1,106 @@
+﻿#  
+# Module manifest for module 'Microsoft.Azure.Commands.AnalysisServices.Dataplane'  
+#  
+# Generated by: Microsoft Corporation  
+#  
```

Comment:
@athipp I will take care of adding this module in AzureRM, so you can ignore this.